draft-ietf-rats-architecture.md

```diff
@@ -994,17 +994,18 @@ Attestation Results may carry a boolean value indicating compliance or non-compl
 The quality of the Attestation Results depend upon the ability of the Verifier to evaluate the Attester.
 Different Attesters have a different _Strength of Function_ {{strengthoffunction}}, which results in the Attestation Results being qualitatively different in strength.
 
-A result that indicates non-compliance can be used by an Attester (in the passport model) or
+An Attestation Result that indicates non-compliance can be used by an Attester (in the passport model) or
 a Relying Party (in the background-check model) to indicate that the Attester
 should not be treated as authorized and may be in need of remediation.  In some cases,
 it may even indicate that the Evidence itself cannot be authenticated as being correct.
 
-An Attestation Result that indicates compliance can be used by a Relying Party to make
-authorization decisions based on the Relying Party's appraisal policy.  The simplest such
-policy might be to simply authorize any party supplying a compliant Attestation Result
-signed by a trusted Verifier.  A more complex policy might also entail comparing information
-provided in the result against Reference Values, or applying more complex logic
-on such information.
+By default, the Relying Party assumes the Attester does not comply with Verifier appraisals. 
+Upon receipt of an authentic Attestation Result and given the Appraisal Policy for Relying Party is satisfied. 
+The Attester is allowed to perform the prescribed actions or access. 
```

Comment:
```suggestion
Upon receipt of an authentic Attestation Result and given the Appraisal Policy for Relying Party is satisfied, then
the Attester is allowed to perform the prescribed actions or access. 
```

Reply:
Better, but "Appraisal Policy for Relying Party" also needs to be "Appraisal Policy for Attestation Results" which is the defined term.

Reply:
One more question: Checking the Attestation Result's authenticity is included in the Appraisal Policy for Attestation Results or not?
I.e., should we call an Attestation Result authentic only after the Appraisal Policy for Attestation Results is satisfied?

Reply:
okay, what wording change do you suggest?